MapboxNavigation/InstructionLabel.swift

```diff
@@ -28,10 +29,28 @@ open class InstructionLabel: StylableLabel, InstructionPresenterDataSource {
             
             let presenter = InstructionPresenter(instruction, dataSource: self, imageRepository: imageRepository, downloadCompletion: update)
             
-            attributedText = presenter.attributedText()
+            let attributed = presenter.attributedText()
+            attributedText = instructionDelegate?.label?(self, willPresent: instruction, as: attributed) ?? attributed
             instructionPresenter = presenter
         }
     }
 
     private var instructionPresenter: InstructionPresenter?
 }
+
+/**
+ The `VoiceControllerDelegate` protocol defines a method that allows an object to customize presented visual instructions.
+ */
+@objc(MBVisualInstructionDelegate)
+public protocol VisualInstructionDelegate: class {
+    
+    /**
+     Called when an InstructionLabel will present a visual instruction.
+     
+     - parameter label: The label that the instruction will be presented on.
+     - parameter instruction: the `VisualInstruction` that will be presented.
+     - parameter presented: the formatted string that is provided by the instruction presenter
+     - returns: optionally, a customized NSAttributedString that will be presented instead of the default, or if nil, the default behavior will be used.
+     */
+    @objc optional func label(_ label: InstructionLabel, willPresent instruction: VisualInstruction, as presented: NSAttributedString) -> NSAttributedString?
```

Comment:
The Objective-C signature of this method is currently `-label:willPresent:as:`. I think it should be `-…:willPresentVisualInstruction:label:presentedAttributedString:` or somesuch.

Reply:
Ah, right you are. Good catch. Will fix.